terraform_compliance/main.py

```diff
@@ -31,6 +31,8 @@ def cli(arghandling=ArgHandling(), argparser=ArgumentParser(prog=__app_name__,
                         help="Directory (or git repository with 'git:' prefix) consists of BDD features", required=True)
     parser.add_argument("--planfile", "-p", dest="plan_file", metavar='plan_file', action=ReadablePlan,
                         help="Plan output file generated by Terraform", required=True)
+    parser.add_argument("--junit", "-j", dest="junit_file", metavar='junit_file', type=str, nargs='?',
```

Comment:
Why do we need `-j` `--junit` parameter here ? `--junit-xml` is already been passed to `radish` which is creating the junit xml file that you need ?

Reply:
I misunderstood the usage. I've now removed this change. Thank you :)